scripts/index.js

```diff
@@ -1 +1,14 @@
-// add your solutions here
\ No newline at end of file
+// add your solutions here
+
+//********postive numbers ********
+
+function noOdds(arr) {
+    let even = arr.filter(val => {
+        return val % 2 === 0;
+    });
+    console.log(even);
```

Comment:
We want to try to return a value from a function. Instead of `console.log(even);` what if we did `return even;`?
Also, nice consistency with your semi-colons.